### DISCUSSION TITLE
ERROR: Connection disallowed: IP address '172.19.0.1' has been dropped. - Nextcloud container logs

### DISCUSSION THREAD

**Discussion by fate8383** [nextcloud/all-in-one#3540] [Questions] [answered]
I am seeing this in the nextcloud docker logs, after latest update. all nextcloud docker containers are part of this subnet, but I have caddy on 172.18.0.0/24 subnet. It was working perfectly fine up until this version and I am not sure where do I touch to see what is blocking it? any help/advice?

**Comment by fate8383** [comment]
Service Unavailable

The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.

I see this on nextcloud instance, direct or via caddy.

**Comment by szaimen** [comment]
Hi, can you show your Caddy config here and also post the compose file of caddy here?

**Comment by fate8383** [comment]
}
https://MYDOMAIN:443 {
    tls /certs/cert.pem /certs/key.pem
    header Strict-Transport-Security max-age=31536000;
    reverse_proxy SERVERIP_192....:11000
}

Caddy file


   environment: # Is needed when using any of the options below
      - APACHE_PORT=11000 # Is needed when running behind a reverse proxy. See https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
      - APACHE_IP_BINDING=0.0.0.0 # Should be set when running behind a reverse proxy that is running on the same host. See https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
 
Nextcloud master compose relevant part


caddy compose  is pretty generic really, but in different docker network, just made it join also nextcloud air network, still not working

**Reply by fate8383** [reply]
AIO login page also behind caddy and it is working:

https://MYDOMAIN:8443 {
    tls /certs/cert.pem /certs/key.pem
    reverse_proxy https://SERVERIP_192...:8080 {
        transport http {
            tls_insecure_skip_verify
        }
    }
}

**Comment by szaimen** [comment]
Can you point caddy at `host.docker.internal:11000` and check if it makes any difference?

**Comment by fate8383** [comment]
> Can you point caddy at `host.docker.internal:11000` and check if it makes any difference?

didn't work, but not seeing 503 anymore just not loading the page

direct server IP
http://192.168...:11000
still gives 503 page

**Comment by fate8383** [comment]
listen.allowed_clients = 127.0.0.1,::1,172.19.0.5,172.19.0.2 could be this perhaps?
.2 is mastercontainer and .5 is apache container but I see .1 is being disallowed and dropped in the nextcloud logs. this is the docker network gateway for nextcloud air network. possible?

**Reply by szaimen** [reply]
Do you still get the same logs and additional ones in the nextcloud container logs?

**Reply by fate8383** [reply]
listen.allowed_clients = 127.0.0.1,::1,172.19.0.8,172.19.0.2
[16-Oct-2023 13:41:38] NOTICE: fpm is running, pid 277
[16-Oct-2023 13:41:38] NOTICE: ready to handle connections
ERROR: Connection disallowed: IP address '172.19.0.1' has been dropped.
ERROR: Connection disallowed: IP address '172.19.0.1' has been dropped.
ERROR: Connection disallowed: IP address '172.19.0.1' has been dropped.
ERROR: Connection disallowed: IP address '172.19.0.1' has been dropped.
ERROR: Connection disallowed: IP address '172.19.0.1' has been dropped.


the same 

apache logs:
{"level":"info","ts":1697456501.6824343,"msg":"using provided configuration","config_file":"/tmp/Caddyfile","config_adapter":""}
[Mon Oct 16 13:41:41.826072 2023] [mpm_event:notice] [pid 53:tid 548069359712] AH00489: Apache/2.4.57 (Unix) configured -- resuming normal operations
[Mon Oct 16 13:41:41.826347 2023] [core:notice] [pid 53:tid 548069359712] AH00094: Command line: '/usr/local/apache2/bin/httpd -D FOREGROUND'
[Mon Oct 16 13:42:27 2023] [error] [client: CLIENTIP] [AH01067: Failed to read FastCGI header] [Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.0 Safari/605.1.15]
[Mon Oct 16 13:42:27 2023] [error] [(104)Connection reset by peer] [client: CLIENTIP] [AH01075: Error dispatching request to :11000: ] [Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.0 Safari/605.1.15]

I don't see host server has 11000 listening, could this be it? I am not sure why though

**Reply by szaimen** [reply]
> I don't see host server has 11000 listening, could this be it? I am not sure why though

Can you post the output of 
```
sudo netstat -tulpn
sudo docker inspect nextcloud-aio-apache
```

**Reply by fate8383** [reply]
tcp        0      0 0.0.0.0:11000           0.0.0.0:*               LISTEN      3122416/docker-prox 
udp        0      0 0.0.0.0:11000           0.0.0.0:*                           3122429/docker-prox 

[
    {
        "Id": "d779391559ec574a33f0b0d02b2e99e36d03abc05e49dcc387fec474cbccae15",
        "Created": "2023-10-16T11:41:02.648979106Z",
        "Path": "/start.sh",
        "Args": [
            "/usr/bin/supervisord",
            "-c",
            "/supervisord.conf"
        ],
        "State": {
            "Status": "running",
            "Running": true,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 3122464,
            "ExitCode": 0,
            "Error": "",
            "StartedAt": "2023-10-16T11:41:03.900207631Z",
            "FinishedAt": "0001-01-01T00:00:00Z",
            "Health": {
                "Status": "healthy",
                "FailingStreak": 0,
                "Log": [
                    {
                        "Start": "2023-10-16T13:53:08.237103313+02:00",
                        "End": "2023-10-16T13:53:08.385580879+02:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2023-10-16T13:53:38.394111302+02:00",
                        "End": "2023-10-16T13:53:38.57986058+02:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2023-10-16T13:54:08.588486172+02:00",
                        "End": "2023-10-16T13:54:08.762939962+02:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2023-10-16T13:54:38.770510677+02:00",
                        "End": "2023-10-16T13:54:38.933470944+02:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2023-10-16T13:55:08.940355997+02:00",
                        "End": "2023-10-16T13:55:09.079590947+02:00",
                        "ExitCode": 0,
                        "Output": ""
                    }
                ]
            }
        },
        "Image": "sha256:3633c37e87a9e35619e50adf33f3e24eb54a24a0099b06b0f9f16c213f90e793",
        "ResolvConfPath": "/var/lib/docker/containers/d779391559ec574a33f0b0d02b2e99e36d03abc05e49dcc387fec474cbccae15/resolv.conf",
        "HostnamePath": "/var/lib/docker/containers/d779391559ec574a33f0b0d02b2e99e36d03abc05e49dcc387fec474cbccae15/hostname",
        "HostsPath": "/var/lib/docker/containers/d779391559ec574a33f0b0d02b2e99e36d03abc05e49dcc387fec474cbccae15/hosts",
        "LogPath": "",
        "Name": "/nextcloud-aio-apache",
        "RestartCount": 0,
        "Driver": "overlay2",
        "Platform": "linux",
        "MountLabel": "",
        "ProcessLabel": "",
        "AppArmorProfile": "",
        "ExecIDs": null,
        "HostConfig": {
            "Binds": [
                "nextcloud_aio_nextcloud:/var/www/html:ro",
                "nextcloud_aio_apache:/mnt/data:rw"
            ],
            "ContainerIDFile": "",
            "LogConfig": {
                "Type": "syslog",
                "Config": {
                    "tag": "docker-{{.Name}}"
                }
            },
            "NetworkMode": "nextcloud-aio",
            "PortBindings": {
                "11000/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "11000"
                    }
                ],
                "11000/udp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "11000"
                    }
                ]
            },
            "RestartPolicy": {
                "Name": "unless-stopped",
                "MaximumRetryCount": 0
            },
            "AutoRemove": false,
            "VolumeDriver": "",
            "VolumesFrom": null,
            "CapAdd": null,
            "CapDrop": [
                "NET_RAW"
            ],
            "CgroupnsMode": "private",
            "Dns": null,
            "DnsOptions": null,
            "DnsSearch": null,
            "ExtraHosts": null,
            "GroupAdd": null,
            "IpcMode": "private",
            "Cgroup": "",
            "Links": null,
            "OomScoreAdj": 0,
            "PidMode": "",
            "Privileged": false,
            "PublishAllPorts": false,
            "ReadonlyRootfs": true,
            "SecurityOpt": null,
            "Tmpfs": {
                "/home/www-data": "",
                "/tmp": "",
                "/usr/local/apache2/logs": "",
                "/var/log/supervisord": "",
                "/var/run/supervisord": ""
            },
            "UTSMode": "",
            "UsernsMode": "",
            "ShmSize": 67108864,
            "Runtime": "runc",
            "ConsoleSize": [
                0,
                0
            ],
            "Isolation": "",
            "CpuShares": 0,
            "Memory": 0,
            "NanoCpus": 0,
            "CgroupParent": "",
            "BlkioWeight": 0,
            "BlkioWeightDevice": null,
            "BlkioDeviceReadBps": null,
            "BlkioDeviceWriteBps": null,
            "BlkioDeviceReadIOps": null,
            "BlkioDeviceWriteIOps": null,
            "CpuPeriod": 0,
            "CpuQuota": 0,
            "CpuRealtimePeriod": 0,
            "CpuRealtimeRuntime": 0,
            "CpusetCpus": "",
            "CpusetMems": "",
            "Devices": null,
            "DeviceCgroupRules": null,
            "DeviceRequests": null,
            "KernelMemory": 0,
            "KernelMemoryTCP": 0,
            "MemoryReservation": 0,
            "MemorySwap": 0,
            "MemorySwappiness": null,
            "OomKillDisable": null,
            "PidsLimit": null,
            "Ulimits": null,
            "CpuCount": 0,
            "CpuPercent": 0,
            "IOMaximumIOps": 0,
            "IOMaximumBandwidth": 0,
            "MaskedPaths": [
                "/proc/asound",
                "/proc/acpi",
                "/proc/kcore",
                "/proc/keys",
                "/proc/latency_stats",
                "/proc/timer_list",
                "/proc/timer_stats",
                "/proc/sched_debug",
                "/proc/scsi",
                "/sys/firmware"
            ],
            "ReadonlyPaths": [
                "/proc/bus",
                "/proc/fs",
                "/proc/irq",
                "/proc/sys",
                "/proc/sysrq-trigger"
            ],
            "Init": true
        },
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/e7b8eb78422f2b5cef3f16851c12c9d904715a1cf9d59bcff4dd065235019f98-init/diff:/var/lib/docker/overlay2/7c6e2f61b35d9f287dc6819496aadaaae6ae0a62ad0b317fe165f5925c9956e4/diff:/var/lib/docker/overlay2/e11544c617371ea1581b56f8de698dd844ae6e19963c65e66befd03fe0168cfd/diff:/var/lib/docker/overlay2/281b3a2de075c469d29ae356bf532095797b91ef2c9111ab6c4a85b2f7fc5c84/diff:/var/lib/docker/overlay2/663dc2cfe102440ca96c22c833ded8ad19aa8ab927b8c0fab37c96e1b078540b/diff:/var/lib/docker/overlay2/1ec011d162d59fd210f261cb40588fa051f786c29be2da882717fae64e65d767/diff:/var/lib/docker/overlay2/fd8e0060b8cb5f0a3c5d85392a60d57086f349f0aac0cb75c433052a0e002b2d/diff:/var/lib/docker/overlay2/3d7cfa64cd788d3c14d38887b37732ad4f069386f3b2e6bc14f0a65ab31cb0ee/diff:/var/lib/docker/overlay2/14b1f14433cf0921dc2619c60534ce95bd61905c619521cb2d2acb82c4607775/diff:/var/lib/docker/overlay2/82c9f5a0df8d4d87dd17e8a42908d3ca322c52e38a92bb77ddb0e6197e5d6113/diff:/var/lib/docker/overlay2/aecfb9d454a23947f3db715b876a4c62f6cad91ec079ad093825f68c07881088/diff:/var/lib/docker/overlay2/090e0a890322806dd14cc48150dce4b6e18ca52a8c58fd3d3e48bbbe680398de/diff:/var/lib/docker/overlay2/2adf61381f766d34d6c8bd50739b45a51e3a94eb786b1015664bb1163cb6e2e7/diff:/var/lib/docker/overlay2/37458dca239559638b41377b71b67251c250bfc2f4c65209512f08b5fd56cbaf/diff",
                "MergedDir": "/var/lib/docker/overlay2/e7b8eb78422f2b5cef3f16851c12c9d904715a1cf9d59bcff4dd065235019f98/merged",
                "UpperDir": "/var/lib/docker/overlay2/e7b8eb78422f2b5cef3f16851c12c9d904715a1cf9d59bcff4dd065235019f98/diff",
                "WorkDir": "/var/lib/docker/overlay2/e7b8eb78422f2b5cef3f16851c12c9d904715a1cf9d59bcff4dd065235019f98/work"
            },
            "Name": "overlay2"
        },
        "Mounts": [
            {
                "Type": "volume",
                "Name": "nextcloud_aio_apache",
                "Source": "/var/lib/docker/volumes/nextcloud_aio_apache/_data",
                "Destination": "/mnt/data",
                "Driver": "local",
                "Mode": "rw",
                "RW": true,
                "Propagation": ""
            },
            {
                "Type": "volume",
                "Name": "nextcloud_aio_nextcloud",
                "Source": "/var/lib/docker/volumes/nextcloud_aio_nextcloud/_data",
                "Destination": "/var/www/html",
                "Driver": "local",
                "Mode": "ro",
                "RW": false,
                "Propagation": ""
            }
        ],
        "Config": {
            "Hostname": "d779391559ec",
            "Domainname": "",
            "User": "www-data",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "ExposedPorts": {
                "11000/tcp": {},
                "11000/udp": {},
                "80/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "NC_DOMAIN=MYDOMAIN",
                "NEXTCLOUD_HOST=nextcloud-aio-nextcloud",
                "COLLABORA_HOST=nextcloud-aio-collabora",
                "TALK_HOST=nextcloud-aio-talk",
                "APACHE_PORT=11000",
                "ONLYOFFICE_HOST=nextcloud-aio-onlyoffice",
                "TZ=Europe/Prague",
                "APACHE_MAX_SIZE=10737418240",
                "APACHE_MAX_TIME=3600",
                "NOTIFY_PUSH_HOST=nextcloud-aio-notify-push",
                "PATH=/usr/local/apache2/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "HTTPD_PREFIX=/usr/local/apache2",
                "HTTPD_VERSION=2.4.57",
                "HTTPD_SHA256=dbccb84aee95e095edfbb81e5eb926ccd24e6ada55dcd83caecb262e5cf94d2a",
                "HTTPD_PATCHES=rewrite-windows-testchar-h.patch 1d5620574fa03b483262dc5b9a66a6906553389952ab5d3070a02f887cc20193"
            ],
            "Cmd": [
                "/usr/bin/supervisord",
                "-c",
                "/supervisord.conf"
            ],
            "Healthcheck": {
                "Test": [
                    "CMD-SHELL",
                    "/healthcheck.sh"
                ]
            },
            "Image": "nextcloud/aio-apache:latest-arm64",
            "Volumes": {
                "/mnt/data": {}
            },
            "WorkingDir": "/usr/local/apache2",
            "Entrypoint": [
                "/start.sh"
            ],
            "OnBuild": null,
            "Labels": {
                "com.centurylinklabs.watchtower.enable": "false"
            },
            "StopSignal": "SIGWINCH"
        },
        "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "4868258b52edccd511646bd0ec926883ef34b6bb542c9d4dd68442cc1cd760c8",
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "Ports": {
                "11000/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "11000"
                    }
                ],
                "11000/udp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "11000"
                    }
                ],
                "80/tcp": null
            },
            "SandboxKey": "/var/run/docker/netns/4868258b52ed",
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "",
            "Gateway": "",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "",
            "IPPrefixLen": 0,
            "IPv6Gateway": "",
            "MacAddress": "",
            "Networks": {
                "nextcloud-aio": {
                    "IPAMConfig": null,
                    "Links": null,
                    "Aliases": [
                        "d779391559ec"
                    ],
                    "NetworkID": "edcedbab7a47f4879d8d341482aa0c05d4fc59c8daa93bc062a992b4d0cb0c62",
                    "EndpointID": "88b0e8fedc2dc5846b2c1ff8b7c30b548f6fc1ae1fd6f141b0c7d945ddca8ea1",
                    "Gateway": "172.19.0.1",
                    "IPAddress": "172.19.0.8",
                    "IPPrefixLen": 16,
                    "IPv6Gateway": "",
                    "GlobalIPv6Address": "",
                    "GlobalIPv6PrefixLen": 0,
                    "MacAddress": "02:42:ac:13:00:08",
                    "DriverOpts": null
                }
            }
        }
    }
]

**Comment by fate8383** [comment]
> Do you still get the same logs and additional ones in the nextcloud container logs?

**Comment by szaimen** [comment]
All right! If you now point caddy at `nextcloud-aio-apache:11000` does it make it work? If not please post the nextcloud container logs again.

**Reply by fate8383** [reply]
No does not work

[16-Oct-2023 14:18:33] NOTICE: fpm is running, pid 277
[16-Oct-2023 14:18:33] NOTICE: ready to handle connections
ERROR: Connection disallowed: IP address '172.19.0.1' has been dropped.
ERROR: Connection disallowed: IP address '172.19.0.1' has been dropped.

**Reply by szaimen** [reply]
So the caddy container is now part of `nextcloud-aio` network and you pointed caddy at `nextcloud-aio-apache` and you still get `ERROR: Connection disallowed: IP address '172.19.0.1' has been dropped.` in the nextcloud container logs?

**Reply by fate8383** [reply]
I added caddy to the AIO network and still not working. logs as attached in the previous reply.

**Reply by fate8383** [reply]
I remove caddy from network page does not load, I add caddy to AIO network I get 503 page.

**Reply by szaimen** [reply]
can yu please show the caddyfile and the docker-compose file for caddy again?

**Reply by fate8383** [reply]
https://MYDOMAIN:443 {
    tls /certs/cert.pem /certs/key.pem
    header Strict-Transport-Security max-age=31536000;
    reverse_proxy nextcloud-aio-apache:11000
}

**Reply by fate8383** [reply]
ERR | ts=1697458871.1459472 logger=http.log.error msg=dial tcp: lookup nextcloud-aio-apache: i/o timeout request={"remote_ip":"CLEINTIP","remote_port":"53883","client_ip":"CLIENTIP","proto":"HTTP/1.1","method":"GET","host":"MYDOMAIN","uri":"/index.php/204","headers":{"Accept-Encoding":["gzip, deflate"],"Accept-Language":["en-US,*"],"Accept":["*/*"],"Connection":["Keep-Alive"],"User-Agent":["Mozilla/5.0 (Macintosh) mirall/3.10.0git (build 17671) (Nextcloud, osx-23.0.0 ClientArchitecture: x86_64 OsArchitecture: x86_64)"],"X-Request-Id":["067c78ef-9b16-45d3-b589-f0ff02385331"],"Cookie":[],"Authorization":[]},"tls":{"resumed":false,"version":772,"cipher_suite":4867,"proto":"","server_name":"MYDOMAIN"}} duration=3.002839716 status=502 err_id=h8tkr6htg err_trace=reverseproxy.statusError (reverseproxy.go:1265)

caddy logs

**Reply by szaimen** [reply]
I have not yet seen your docker-compose file for caddy. can you post it here?

**Reply by fate8383** [reply]
```
networks:
  mydockernetwork:
    external: true
services:
  caddy:
    image: caddy:latest
    restart: always
    container_name: caddy
    volumes:
      - /home/pi/caddy/Caddyfile:/etc/caddy/Caddyfile
      - /home/pi/caddy/certs:/certs
      - /home/pi/caddy/config:/config
      - /home/pi/caddy/data:/data
      - /home/pi/caddy/sites:/srv
    networks:
      - mydockernetwork
    ports:
      - 443:443
      - 8443:8443 #aio
```

**Reply by szaimen** [reply]
So apparently is it not part of the `nextcloud-aio` netowork. How should it reach the nextcloud-aio-apache container then?

**Reply by fate8383** [reply]
it does reach to the network because I added manually to AIO network as well. I would not be able to see 
Service Unavailable

The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.

page otherwise, without being part of the network it does not load the page at all.

**Reply by fate8383** [reply]
/srv # ping nextcloud-aio-apache
PING nextcloud-aio-apache (172.19.0.8): 56 data bytes
64 bytes from 172.19.0.8: seq=0 ttl=64 time=0.244 ms
64 bytes from 172.19.0.8: seq=1 ttl=64 time=0.242 ms
64 bytes from 172.19.0.8: seq=2 ttl=64 time=0.243 ms
^C
--- nextcloud-aio-apache ping statistics ---
3 packets transmitted, 3 packets received, 0% packet loss
round-trip min/avg/max = 0.242/0.243/0.244 ms

from caddy container

**Reply by szaimen** [reply]
So basically you say you are note able to add the caddy container to the `nextcloud-aio` network?

**Reply by fate8383** [reply]
I am saying the opposite :) I have caddy by default on my docker network where it servers for many other services, but I manually added caddy to AIO network as well. Which I didn't have to do is up until now, because I bind AIO apache to 0.0.0.0 of the docker server host and caddy can reach that via 192.168... now this is broken. 

But most importantly Nextcloud returns 503 regardless of caddy, why? and the logs only says dropped AIO network gateway IP.

**Reply by szaimen** [reply]
> But most importantly Nextcloud returns 503 regardless of caddy, why? and the logs only says dropped AIO network gateway IP.

yes this is weird. I dont have an explanation for this currently.

**Reply by fate8383** [reply]
now I reverted all back and caddy is fine actually as it was before. Connectivity is fine in other words. Problem is really this 503 error, this is what Nextcloud returns and I don't know where to look for why.

**Reply by szaimen** [reply]
Can you for a test enable ipv6 for the nextcloud-aio network and check if it makes any difference?
See https://github.com/nextcloud/all-in-one/discussions/2045

**Reply by fate8383** [reply]
> Can you for a test enable ipv6 for the nextcloud-aio network and check if it makes any difference?
> 
> See https://github.com/nextcloud/all-in-one/discussions/2045

I am not comfortable doing this, as many other dockers I have all will be impacted by this. I am not ipv6 savvy:) any other solutions?

**Reply by szaimen** [reply]
I am wondering if this is actually leading to your issue (having ipv6 enabled partially and not completrly)

**Reply by fate8383** [reply]
I am not why ipv6 has anything to do with this. I see the disallowed ip is ip4 and I tried to add this to the config php then restarting continers I don't see Nextcloud respects that line although it is there...so still drops that ip. Is there a way to roll back to the previous version?

**Reply by szaimen** [reply]
> I don't see Nextcloud respects that line although it is there

Which is another clue that this is not thw underlying issue

**Reply by fate8383** [reply]
I meant Nextcloud logs does not show it but when I go to container cli and I see it in the config. But yeah I am not sure why this is happening all of a sudden. Is there an easy way to go back previous version?

**Comment by fate8383** [comment]
listen.allowed_clients = 127.0.0.1,::1,172.19.0.7,172.19.0.2
I would like to add here dropping IP as well, is that possible? I am suspecting this is the issue.

**Comment by CptnNoFuture** [comment]
I've been having exactly the same problem since I updated to v7.4.1. I tried activating IPv6 for docker as suggested. Sadly it didn't change anything except for some new IPv6-entries under "listen.allowed_clients".
The release-notes for v7.4.1 mention a fix for a problem with "listen.allowed_clients". Maybe this new and odd behaviour has to do with the latest code-changes in "Containers/nextcloud/start.sh"?

**Reply by szaimen** [reply]
Hi, can you post your reverse proxy config, the docker-compose file or docker run command of the reverse proxy, the output of `sudo docker inspect nextcloud-aio-apache` and the output of `sudo docker logs nextcloud-aio-nextcloud` here? Thanks!

**Reply by CptnNoFuture** [reply]
I'm using Nginx Proxy Manager on another Server within the same LAN. It's not within the docker-network of nextcloud. It directs to the IP of the nextcloud-server and uses the exposed port 11000 of the apache-docker container. Here is the docker compose file:
[docker-compose.yml.txt](https://github.com/nextcloud/all-in-one/files/13046258/docker-compose.yml.txt)

And here is the configuration for NPM:

[NPM1](https://github.com/nextcloud/all-in-one/assets/88511034/5b83f6a0-87ac-49c2-a53f-b56ec73239b0)
[NPM2](https://github.com/nextcloud/all-in-one/assets/88511034/9e0f6962-c53e-486b-ac6a-52979a2e193e)
[NPM3](https://github.com/nextcloud/all-in-one/assets/88511034/330ef068-8f6c-4b9d-884f-f98f5b6e33ba)

Last not least, here are the results from the two commands:
[sudo docker inspect nextcloud-aio-apache.txt](https://github.com/nextcloud/all-in-one/files/13046183/sudo.docker.inspect.nextcloud-aio-apache.txt)
[sudo docker logs nextcloud-aio-nextcloud.txt](https://github.com/nextcloud/all-in-one/files/13046200/sudo.docker.logs.nextcloud-aio-nextcloud.txt)

**Reply by szaimen** [reply]
Thanks! Can you post the output of `sudo docker info` ?

**Reply by szaimen** [reply]
Also why did you cover the `Forward Hostname / IP` in the screenshot? is this a public ip? I need to know what is in there...

**Reply by CptnNoFuture** [reply]
`Forward Hostname / IP` points to 192.168.178.54 within my local network. Sorry, I covered it out of habit.
Here are the results of `sudo docker info`:
[sudo docker info.txt](https://github.com/nextcloud/all-in-one/files/13046732/sudo.docker.info.txt)

**Reply by szaimen** [reply]
@Zoey2936 do you have any idea why the nextcloud container thinks that the request comes from the gatway as the request in the end should still be coming from the apache container. So this is really weird...

**Reply by Zoey2936** [reply]
sorry, no idea

**Reply by szaimen** [reply]
All right, still thanks!

@CptnNoFuture can you show your daemon.json file of docker?

**Reply by CptnNoFuture** [reply]
{
  "data-root": "/var/lib/docker",
  "ipv6": true,
  "fixed-cidr-v6": "fd12:3456:789a:1::/64",
  "experimental": true,
  "ip6tables": true
}

**Reply by szaimen** [reply]
looks good. So I am out of ideas why this is happening. 

Can you run 
```
# log into container
sudo docker exec -it nextcloud-aio-nextcloud bash
```
```
# now inside container
apk add nano
# edit start.sh with nano
nano /start.sh
```
Then edit the following line: https://github.com/nextcloud/all-in-one/blob/499dccb12255304b3124c4f787be169b053dad2e/Containers/nextcloud/start.sh#L144
and add `172.20.0.1` as additionally allowed value and save the file with `[ctrl]+[o]` -> `[enter]` and `[ctrl]+[x]` to close it.

Then log out from the container via `exit` and run `sudo docker restart nextcloud-aio-nextcloud`. 

Afterwards please check if it makes it work. If not, please post the nextcloud container logs again.

**Reply by szaimen** [reply]
@fate8383 @CptnNoFuture If the above makes it work for you, I am open for creating a PR that does this automatically. So I would need some feedback on this first

**Reply by CptnNoFuture** [reply]
For me that workaround did the trick. Everything is working again.

**Reply by szaimen** [reply]
weird but glad that it resolves things.

Let me create a PR

**Reply by fate8383** [reply]
I was saying "allow.clients" is the issue. I had to move to another image, sorry I can't confirm whether that would work for me or not. But I bet it would. Thanks

**Reply by szaimen** [reply]
PR here: https://github.com/nextcloud/all-in-one/pull/3574

**Reply by szaimen** [reply]
> I was saying "allow.clients" is the issue

Yes I saw it but it still doesnt make sense. I am now just fixing this weird behaviour but I consider it rather to be a workaround for that...

**Reply by CptnNoFuture** [reply]
Thanks for your help.

**Comment by szaimen** [comment, marked as answer]
This is now fixed with v7.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by CptnNoFuture** [reply]
The new version works for me. Thanks again!

**Reply by szaimen** [reply]
Great! Thanks for verifying!

**Comment by tullsta2** [comment]
Hello,

not sure whether I see the same issue as I just installed aio, but I also get a
> ERROR: Connection disallowed: IP address '172.25.0.1' has been dropped.

and a

> Service Unavailable
> The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.

Initially I thought it might just take a little while for e.g. databaase-setup, but 2 hours later and after reading through this conversation, I moved the master container to the nextcloud/all-in-one:beta (7.50beta) channel, but this did not resolve anything for me.

Happy to provide more debug data - just let me know.

Thanks and kind regards - tullsta

**Reply by szaimen** [reply]
Hi, can you post your reverse proxy config, the docker-compose file or docker run command of the reverse proxy, the output of `sudo docker inspect nextcloud-aio-apache` and the output of `sudo docker logs nextcloud-aio-nextcloud` here? Thanks!

**Reply by tullsta2** [reply]
Hi, a few minues ago the AIO GUI said that there are updates for containers available.

So, after the update of the containers things look better:
 - no error connection diallowed anymore
 - Nextcloud AIO 7.5.0 working (NC7.1.2 running)

will now start looking into the details.

THanks + kind regards - Volker

**Reply by szaimen** [reply]
> So, after the update of the containers things look better:
> 
> * no error connection diallowed anymore
> * Nextcloud AIO 7.5.0 working (NC7.1.2 running)

Good to hear!

**Reply by szaimen** [reply]
If it works, I dont need any more information about your setup
